CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+# Changelog
+
+## 6.2.6
+
+* Package Redis `v6.2.6`.
+* Add Redis Sentinel helper scripts.
```

Comment:
Probably should have been more descriptive. Something like "Add Redis Sentinel helper scripts for running on Kubernetes". 🙈

Reply:
Improved in https://github.com/Intellection/docker-redis/pull/2.